cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
You need to register your Movie model here to make it accessible through the Django admin interface. Use the admin.site.register() function to register your model.